src/payment/strategies/square/square-payment-strategy.ts

```diff
@@ -158,16 +167,23 @@ export default class SquarePaymentStrategy implements PaymentStrategy {
         };
     }
 
+    private _getInitializeOptions(): SquarePaymentInitializeOptions {
+        if (!this._squareOptions) {
+            throw new NotInitializedError(NotInitializedErrorType.PaymentNotInitialized);
+        }
+
+        return this._squareOptions;
+    }
+
     private _handleWalletNonceResponse(
-        errors?: NonceGenerationError[],
+        errors?: NonceGenerationError[] | undefined,
```

Comment:
If the errors field is optional why do we need to add a union type of undefined?